synapse/config/federation.py

```diff
@@ -69,14 +69,20 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         # when trying to reach an unavailable destination.
         # Unlike previous configuration those values applies across
         # multiple requests and the state of the backoff is stored on DB.
-        self.destination_min_retry_interval_ms = Config.parse_duration(
-            federation_config.get("destination_min_retry_interval", "10m")
+        self.destination_min_retry_interval_ms = min(
+            Config.parse_duration(
+                federation_config.get("destination_min_retry_interval", "10m")
+            ),
+            2**62,
         )
```

Comment:
I don't think this one needs to have a minimum set since we'll check against `destination_max_retry_interval_ms` regardless.